nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-ui/src/main/webapp/css/connection-configuration.css

```diff
@@ -122,4 +122,16 @@ div.undefined {
     line-height: 32px;
     height: 32px;
     padding: 0px 10px;
-}
\ No newline at end of file
+}
+
+.multi-column-settings {
```

Comment:
I don't think we need to duplicate this styles here. The styles from the `connection-details.css` should be in scope.